src/client/game/game.js

```diff
@@ -1,9 +1,15 @@
 import React, { Component } from 'react';
 
-export default function Game(props) {
-    return (
-        <div className="text-center">
-            <h3>THIS IS GAME</h3>
-        </div>
-    );
+import style from '../index.scss';
```

Comment:
Might be time to rename index.scss to common.scss, looking weird here. Index should be used in index.js or app.js and should be used to do general styling. Classes and ids should be moved to this common.scss :+1: